.github/workflows/changeset-ai-releases.yml

```diff
@@ -109,7 +104,7 @@ jobs:
       # Generate release notes using OpenAI if not already edited, GITHUB_OUTPUT: 'RELEASE_NOTES' and 'OPENAI_PROMPT'
       - name: AI Release Notes
         if: ${{ !contains(github.event.pull_request.labels.*.name, 'openai-edited') }}
-        uses: RooVetGit/Roo-GHA/.github/actions/ai-release-notes@main
+        uses: ./.github/actions/ai-release-notes
         id: ai_release_notes
         with:
           GHA_PAT: ${{ secrets.CROSS_REPO_ACCESS_TOKEN }}
```

Comment:
Do we still need this token here?

Reply:
yeah, that's the only way to pass secrets to a composite action